src/components/calculator.js

```diff
@@ -0,0 +1,41 @@
+const Calculator = () => (
+  <div className="calApp">
+    <div className="renderResult">
+      <div className="result">0</div>
+    </div>
+    <div className="calInputs">
+      <div className="inputs">
+        <input type="button" value="AC" />
+        <input type="button" value="+/-" />
+        <input type="button" value="%" />
+        <input type="button" className="mathSigns" value="/" />
+      </div>
+      <div className="inputs">
+        <input type="button" value="7" />
+        <input type="button" value="8" />
+        <input type="button" value="9" />
+        <input type="button" className="mathSigns" value="x" />
+      </div>
+      <div className="inputs">
+        <input type="button" value="4" />
+        <input type="button" value="5" />
+        <input type="button" value="6" />
+        <input type="button" className="mathSigns" value="-" />
```

Comment:
As described [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Create%20at%20least%201%20child%20component) this component is expected to have at least one child component, in order words, I recommend implementing this to ensure that you meet up the requirements for this milestone.

HINT: Whenever you find yourself repeating same line of code, then its time to create a component for that logic. I think the buttons will be a good candidate for this (child component)

Reply:
@DuaneDave 
 thanks for your kind review, I would implement the required change, however, I would like you to explain in clearer terms what you want me to do in the second required change because, at this stage of the project, it was stated that no logic is required.